scripts/export-rmib-onnx.py

```
@@ -0,0 +1,215 @@
+    q[:, 3] = 1.0
+    for ci, ch in enumerate(order):
+        q = qmul(q, axisq(rad[:, ci], axis_of[ch]))
+    return q
+
+
+def preprocess(bvh_dir):
+    from bvh import Bvh
+    files = sorted(glob.glob(os.path.join(bvh_dir, "**/*.bvh"), recursive=True))
+    print(f"found {len(files)} BVH files")
+    S, E, I = [], [], []
+    for i, path in enumerate(files):
```

Ruff (0.15.18)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)